src/main/kotlin/org/rust/ide/newProject/Utils.kt

```diff
@@ -48,6 +48,7 @@ fun Project.makeDefaultRunConfiguration(template: RsProjectTemplate) {
     val configuration = when (template) {
         is RsGenericTemplate.CargoBinaryTemplate -> configurationFactory.createCargoRunConfiguration()
         is RsGenericTemplate.CargoLibraryTemplate -> configurationFactory.createCargoTestConfiguration()
+        is RsCustomTemplate.ProcMacroTemplate -> configurationFactory.createCargoTestConfiguration()
```

Comment:
Note that a project generated with `ProcMacroTemplate` doesn't contain any tests so this run configuration will be useful only after a user adds `tests` module. So I'm not sure if we really should create a run configuration here

Reply:
Well, ok, let's remove it for now